src/main/rx/xbus.c

```diff
@@ -158,9 +181,68 @@ static void xBusUnpackFrame(void)
 
 }
 
+static void xBusUnpackRJ01Frame(void)
+{
+    // Calculate the CRC of the incoming frame
+    uint16_t crc = 0;
+    uint16_t inCrc = 0;
+    uint8_t i = 0;
+    uint16_t value;
+    uint8_t frameAddr;
+
+    // When using the Align RJ01 receiver with 
+    // a MODE B setting in the radio (XG14 tested)
+    // the MODE_B -frame is packed within some
+    // at the moment unknown bytes before and after:
+    // 0xA1 __ __ 0xA1 12*(High + Low) CRC1 CRC2 + __ __ __
```

Comment:
I have done some more measurements and what I can see it always starts with the same byte sequence: 0xA1 0x1E 0x06 0xA1 ....
Where the second 0xA1 is the start of the MODE B package. My guess is that the trailing characters are some kind of checksum of the total package. I have not bothered to try and decode them as there is a checksum for the channel data in the MODE B package. Hence I think this is robust enough for usage.

I attached a small image from the logic analyzer. The red circles are the check sums for the inner package, i.e. the MODE B package. The last three bytes, blue circle, is probably some kind of check sum for the total package. I have not tried to decode/find a solution for these. The first row shows the delay between the packages and the second row shows the start of the sequence.
![rj01_mode_b](https://cloud.githubusercontent.com/assets/6293653/5605599/4591e218-9404-11e4-978c-33d6516f1bfa.png)